addons/pos_restaurant/static/src/app/screens/receipt_screen/order_receipt/order_receipt.xml

```diff
@@ -2,33 +2,28 @@
 <templates id="template" xml:space="preserve">
 
     <t t-name="pos_restaurant.OrderReceipt" t-inherit="point_of_sale.OrderReceipt" t-inherit-mode="extension">
-        <xpath expr="//ReceiptHeader" position="after">
-            <div t-if="props.data.isBill" class="fs-2 mb-1">Pro forma receipt</div>
-        </xpath>
-        <xpath expr="//div[hasclass('receipt-change')]" position="attributes">
-            <attribute name="t-if">'order_change' in taxTotals and !props.data.isBill</attribute>
-        </xpath>
```

Comment:
is this moved somewhere else?

Reply:
I add it back, now is a proforma if the order isn't finalized